indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -438,20 +454,70 @@ function EditableListDisplay({
       />
     );
   };
+  const renderTags = (__, editable) => {
+    const tags = editable.tags.sort((a, b) => a.code.localeCompare(b.code));
+    const labelTags = () => {
+      return (
+        <div style={{display: 'flex', flexWrap: 'nowrap'}}>
+          {tags.map(t => {
+            return (
+              <Label key={t.code} size="small" color={t.color}>
+                {t.code}
+              </Label>
+            );
+          })}
+        </div>
+      );
+    };
+
+    const colorTags = () => {
+      return (
+        <div>
+          {tags.map(t => {
+            return <Label key={`${t.code}-dot`} circular empty color={t.color} />;
```

Comment:
I think this is enough:
```suggestion
            return <Label key={t.code} circular empty color={t.color} />;
```